repo-infra/verify/verify-boilerplate.sh

```diff
@@ -38,7 +38,7 @@ cleanup() {
 }
 
 pushd "${boilerDir}" >/dev/null
-if ! python -m unittest boilerplate_test 2>"${unitTestOut}"; then
+if ! python3 -m unittest boilerplate_test 2>"${unitTestOut}"; then
```

Comment:
You may need to change this to directly use python 3: https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/blob/017c935a0c11cf9e572345600a80733dbf851081/repo-infra/verify/boilerplate/boilerplate.py#L1

Or we need to update the container image used for the job to install "python-is-python3": https://askubuntu.com/questions/942930/usr-bin-env-python-no-such-file-or-directory

Reply:
I tried change this to python3, and did ``make verify`` locally, it still failed with:

```
/usr/bin/env: ‘python’: No such file or directory
make: *** [Makefile:25: verify] Error 127
```

Reply:
The job definition is:

```
  - name: pull-sig-storage-lib-external-provisioner-unit
    always_run: true
    decorate: true
    path_alias: sigs.k8s.io/sig-storage-lib-external-provisioner
    annotations:
      testgrid-dashboards: sig-storage-lib-external-provisioner
      testgrid-tab-name: unit
      description: Unit tests in sig-storage-lib-external-provisioner repo.
    spec:
      containers:
      - image: golang:1.19
        command:
        - make
        args:
        - test
```